board/piksiv3/rootfs-overlay/etc/init.d/S86_mmc

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+name="mmc_driver"
+
+start() {
+  modprobe mmc_core
```

Comment:
These don't seem to be the correct module names-- in S75dev_boot we run `modprobe sdhci_of_arasan`, copy the .elf, .bit, and .ini then unmount and remove the sdhci_of_arason module.

Reply:
Never mind, didn't notice this PR was closed...